softwarereview_editor_management.pt.Rmd

```diff
@@ -0,0 +1,102 @@
+
+- Inicie um canal privado para discussão (para que você não tenha um histórico no canal de editores no qual futuros editores entrarão, o que pode ser incômodo).
+
+- Faça ping nos editores para garantir que eles recebam uma notificação, pois esse é um tópico importante.
+
+- Espere que a maioria dos editores se manifeste antes de convidar alguém. Dê a eles uma semana para responder.
+
+## Convidando um novo editor {#inviting-a-new-editor}
+
+- Os candidatos podem começar por ser [editores convidados](#guesteditor).
+  Ao convidá-los como editores convidados, convide-os da mesma forma que você convidaria um editor convidado por outros motivos.
+
+- Se um candidato for o primeiro editor convidado, avalie como foi o processo após o envio. Peça novamente a opinião de outros editores.
+
+- Envie um e-mail.
+
+```
+We would like to invite you to join the rOpenSci editorial board as a full member. [SPECIFIC REASONS FOR INVITATION (MENTION CONTRIBUTIONS TO ROPENSCI)]. 
+We think you would make a wonderful addition to the team.
+
+[IF GUEST EDITOR:You are familiar with the editor's role as you've been a guest editor].  We aim for editors to handle four packages per year ([IF GUEST EDITOR including the one one you just finished!]).  
+We ask that editors make an informal commitment of serving for two years, and re-evaluate their participation after that.  
+On a short-term basis, any editor can decline to handle a package or say, "I'm pretty busy, I can't handle a new package for a few weeks."
+
+In addition to handling packages, editors weigh in on group editorial decisions, such as whether a package is in-scope, and determining updates to our policies. 
+We generally do this through Slack, which we expect editors to be able to check regularly. 
+We have editorial board calls annually.  
+We also rotate Editor-in-Chief responsibilities (first-pass scope decisions and assigning editors) amongst the board about quarterly. 
+You'll have the opportunity to enter this rotation once you have been on the board for some time, usually at least six months. 
+Some of us also take on bigger projects to improve the peer-review process, though this is entirely optional. 
+
+We hope that you'll join the board!  
+It's an exciting time for peer-review at rOpenSci.
+
+Please give this some thought, ask us any questions you have, and let us know whether you can join us.
+
+Best,
+[EDITOR], on behalf of the rOpenSci Editorial Board
+```
+
+## Integração de um novo editor {#onboarding-a-new-editor}
+
+- Informe o gerente da comunidade rOpenSci para que
+  
+  - editores sejam adicionados à lista de [site da rOpenSci](https://github.com/ropensci/roweb3/#team-member).
+  - Você pode criar uma postagem de introdução no blog.
+
+- Se eles ainda não o fizeram como editores convidados, solicite que o novo editor ative o [a autenticação de dois fatores (2FA) para o GitHub](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa).
```

Comment:
sugestão: trocar 'o' por 'a' antes da referencia a autenticacao e remover 'a' dentro do []

Reply:
Suggestions included in the review.